src/lib/api-client.js

```diff
@@ -205,6 +205,9 @@ export function deleteItem(address){
         itemAddress: address
     }).then(data => {
         Dispatcher.handleAction('ITEMS_RECEIVED', data);
+        Dispatcher.handleAction('ITEM_DELETED', {
```

Comment:
Instead of creating two actions at once, have the ItemStore listen for ITEM_DELTED as well. (see https://github.com/TheFourFifths/consus-client/pull/38#pullrequestreview-11621592)